src/lib/y2network/interface_config_builders/bonding.rb

```diff
@@ -118,6 +119,16 @@ def bondable?(iface)
 
         true
       end
+
+      # Convenience method to check whether the config of an enslaved interface
+      # is valid or not
+      #
+      # @param iface [String] slave name to be validated
+      def valid_slave_config?(iface)
```

Comment:
I am not sure whether this logic belongs here. Please, make a note and discuss later, when we have better support for validation/errors reporting.